scripts/install-rkt.sh

```diff
@@ -29,6 +29,16 @@ tar xvzf rkt-v"${version}".tar.gz
 cat <<EOF >install-pak
 #!/bin/bash
 
+# fix mkdir issues with checkinstall and fstrans
+for dir in /usr/lib/rkt/stage1-images/\\
+        /usr/share/man/man1/\\
+        /usr/share/bash-completion/completions/\\
+        /usr/lib/tmpfiles.d/\\
+        /usr/lib/systemd/system/
+do
+    mkdir -p \$dir 2>/dev/null || :
```

Comment:
#3123 is where I come from.
I know what does `|| :` but
* this diff does not include `-e` in shebang
* `mkdir -p` will fail only if there was fs problem (lack of inodes, fs mounted in ro)
so `|| :` seems to be remainder of testing process, which should be removed before pushing commit to GH.

It's only a nit. I only wanted to ensure if there is another reason why this construct is used here.